lxd/daemon.go

```diff
@@ -1664,10 +1664,10 @@ func (d *Daemon) Stop(ctx context.Context, sig os.Signal) error {
 		// Full shutdown requested.
 		if sig == unix.SIGPWR {
 			logger.Info("Stopping instances")
-			instancesShutdown(s, instances)
+			_ = instancesShutdown(s, instances)
 
 			logger.Info("Stopping networks")
-			networkShutdown(s)
+			_ = networkShutdown(s)
```

Comment:
networksShutdown looks like it should be made similar to `instancesShutdown` in that it doesn't return an error and logs any errors as warnings.